src/azure-cli/azure/cli/command_modules/keyvault/_help.py

```diff
@@ -143,34 +163,171 @@
 
 helps['keyvault delete'] = """
 type: command
-short-summary: Delete a key vault.
+short-summary: Delete a Vault or HSM.
 examples:
   - name: Delete a key vault. (autogenerated)
     text: |
         az keyvault delete --name MyKeyVault --resource-group MyResourceGroup
     crafted: true
 """
 
+helps['keyvault list-deleted'] = """
+type: command
+short-summary: Gets information about the deleted Vaults or HSMs in a subscription.
+"""
+
+helps['keyvault purge'] = """
+type: command
+short-summary: Permanently deletes the specified Vault or HSM. Aka Purges the deleted Vault or HSM.
+"""
+
 helps['keyvault key'] = """
 type: group
 short-summary: Manage keys.
 """
 
+helps['keyvault key delete'] = """
+type: command
+short-summary: Deletes a key of any type from storage in Vault or HSM.
```

Comment:
Shall we use the first person voice, right?
```suggestion
short-summary: Delete a key of any type from storage in Vault or HSM.
```